packages/x-components/tests/e2e/cucumber/spellcheck.spec.ts

```diff
@@ -1,13 +1,20 @@
 import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { SearchResponse } from '@empathyco/x-adapter';
 
 Given('a results API response for a misspelled word', () => {
   cy.intercept('https://api.empathy.co/search', req => {
-    req.reply({
+    req.reply(<SearchResponse>{
```

Comment:
I think in that case we can remove all the response expects the spellcheck itself.

Reply:
The `SearchResponse` interface says that every property is required. That´s what I´m doing in this PR, ensuring that these interfaces are fulfilled. A little bird has told me that the great @javieri-empathy wants to relax this interfaces, and make every property optional. After that relaxation I agree with your idea makes sense and can be done. However not right now.

Reply:
What I meant is that you're mocking a huge `SearchResponse` when you only need the spellcheck in the spellcheck tests.

Maybe we can use a `Partial` here? I don't know man but for me this reply has more note than info.

Whatever.